platform/windows/display_server_windows.cpp

```diff
@@ -3126,14 +3128,6 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 #endif
 
 		RegisterTouchWindow(wd.hWnd, 0);
-
-		TRACKMOUSEEVENT tme;
-		tme.cbSize = sizeof(TRACKMOUSEEVENT);
-		tme.dwFlags = TME_LEAVE;
-		tme.hwndTrack = wd.hWnd;
-		tme.dwHoverTime = HOVER_DEFAULT;
-		TrackMouseEvent(&tme);
```

Comment:
Just to be sure, this removal (without replacing by `track_mouse_leave_event`) was intentional?

Reply:
I think it's useless to call it twice since it is called at the WndProc/WM_ACTIVATE event now